src/Microsoft.Azure.SignalR.Protocols/MulticastDataMessage.cs

```diff
@@ -141,6 +141,16 @@ public class GroupBroadcastDataMessage : MulticastDataMessage
         /// </summary>
         public IReadOnlyList<string> ExcludedList { get; set; }
 
+        /// <summary>
+        /// Gets or sets the list of excluded user Ids.
+        /// </summary>
+        public IReadOnlyList<string> ExcludedUserList { get; set; }
```

Comment:
should we support exclude connection id and user id at the same time?
if not, should we add an ExcludeTarget (connection id/user id) instead?

Reply:
I think we can support both at the same time